ui/ui-components/components/Loader.tsx

```diff
@@ -1,6 +1,11 @@
-import * as React from "react";
-import { Spinner } from "react-bootstrap";
+import { Spinner, SpinnerProps } from "react-bootstrap";
 
-export default function Loader({ size }: { size?: "sm" }) {
-  return <Spinner animation="grow" role="status" size={size} />;
+interface Props extends React.HTMLAttributes<HTMLElement> {
+  size?: SpinnerProps["size"];
 }
+
+const Loader: React.FC<Props> = ({ size, ...props }) => (
+  <Spinner {...props} animation="border" role="status" size={size} />
```

Comment:
```suggestion
const Loader: React.FC<Props> = ({ props }) => (
  <Spinner {...props} animation="border" role="status" />
```
No need to explicitly assign size.